File: 프로그래머스/lv0/120849. 모음 제거/모음 제거.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        char[] vowels = {'a', 'e', 'i', 'o', 'u'};
         for (char c : my_string.toCharArray()) {
            boolean isVowel = false;
            for (char vowel : vowels) {
                if (c == vowel) {
                    isVowel = true;
                    break;
                }
            }
            
            if (!isVowel) {
                answer += c;
            }
        }
        return answer;
    }
}

File: 프로그래머스/unrated/120849. 모음 제거/모음 제거.py
def solution(my_string):
    answer = ''
    vowels = ['a', 'e', 'i', 'o', 'u']
    
    for c in my_string:
        is_vowel = False
        
        for vowel in vowels:
            if c == vowel:
                is_vowel = True
                break
        
        if not is_vowel:
            answer += c
    
    return answer